.release/docker/dockerfile

```diff
@@ -0,0 +1,84 @@
+#This is the docker official image
```

Comment:
@mdeggies yeah we could move it to an official dir, the only problem is that the entrypoint.sh would need to be in the same directory - because the official dir would become the [docker context](https://github.com/docker-library/official-images#instruction-format) (due to the way the official images are built). But we want to share the entrypoint.sh between both the dockerfiles so this would mean that we'd have to reference `.release/official/entrypoint.sh` or `official/entrypoint.sh` from the CRT dockerfile in the root of the repo. But maybe that would be the lesser evil? 🤔

Reply:
You could also symlink it from one subdirectory to another